package/kedro_viz/launchers/cli.py

```diff
@@ -270,3 +270,25 @@ def deploy(region, bucket_name):
         )
     finally:
         viz_deploy_timer.terminate()
+
+
+@viz.command(context_settings={"help_option_names": ["-h", "--help"]})
+def build():
+    """Create build directory of local Kedro Viz instance with static data"""
+
+    try:
+        load_and_populate_data(Path.cwd(), ignore_plugins=True)
+        deployer = DeployerFactory.create_deployer("local")
+        url = deployer.deploy_and_get_url()
+
+        click.echo(
+            click.style(
+                "\u2728 Success! Kedro-Viz build files have been successfully added to the "
+                f"{url} directory.",
+                fg="green",
+            )
+        )
+
+    except Exception as ex:
+        traceback.print_exc()
```

Comment:
What kind of information will show up in this traceback? Is that easy to read/understand for users?

Reply:
Yes, It won't be easy to read and understand for users. I updated the Exception part, Thanks.